### PR TITLE
Updated para.json

### PR DESCRIPTION
**Pull request by RajuKalidindi**

### CODE REVIEW

**Comment by ayush1999** [comment]
Looks like a `syntaxerror` causes the build to fail. @RajuKalidindi (https://travis-ci.org/p-society/typeracer-cli/builds/377360975?utm_source=github_status&utm_medium=notification)

**Comment by knrt10** [comment]
@RajuKalidindi please run npm test before opening PR. It checks all the syntax.

File: paragraphs/para.json [hunk 1]
```diff
@@ -7362,5 +7362,121 @@
...
+  {
+  "id": 1848,
+  "para": "Love means never having to say you're sorry."
+ },
+  {
+  "id": 1849,
+  "para": "Go ahead, make my day."
+ },
+  {
+  "id": 1850
```
Comment (knrt10) [inline paragraphs/para.json]: You forgot a comma here.

**Comment by IamRaviTejaG** [comment]
Good work @RajuKalidindi. Merging now!

**Pull request merged**